modules/core/client/app/init.js

```diff
@@ -19,7 +19,7 @@ angular.module(ApplicationConfiguration.applicationModuleName).run(function ($ro
     if (toState.data && toState.data.roles && toState.data.roles.length > 0) {
       var allowed = false;
       toState.data.roles.forEach(function (role) {
```

Comment:
I don't think lodash is a dependency right now, and wouldn't want to introduce it for this case.  Can use Array.prototype.some() but not sure about browser support issues.

Probably best to just change to standard for loop, though it's not as elegant :)